python05/quiz_app.py

```diff
@@ -21,19 +21,21 @@ def wrapper_timer(*args, **kwargs):
     return wrapper_timer
 
 
-def create_questions_list(data) -> list:
+def create_questions_list(questions_from_section) -> list:
     """This function creates the list of questions after opening the file with questions
     data in the main block."""
     questions = []
-    for line in data.read().splitlines():
+    for line in questions_from_section.read().splitlines():
         questions.append(line)
     return questions
 
 
 @timer
 def start_quiz(questions) -> list:
-    """After the user answers, function creates the list which will contain data with each
-    question and user answers."""
+    """
+    After the user answers, function creates the list which will contain data with each
+    question and user answers.
+    """
```

Comment:
PEP 257 says "The closing quotes are on the same line as the opening quotes. This looks better for one-liners."
[https://www.python.org/dev/peps/pep-0257/#one-line-docstrings](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings)

I don't know what the convention is, so it's up to you :)

Reply:
Yes, but this docstring has two lines. So i think quotes must be on a separate lines.